src/commands/external.rs

```diff
@@ -50,7 +51,7 @@ pub async fn execute_external_subcommand(
         }
         Err(Error::NotFound(e)) => {
             tracing::debug!("Tried to resolve {plugin_name} to plugin, got {e}");
-            eprintln!("Error: '{plugin_name}' is not a known Spin command. See spin --help.\n");
+            error!("'{plugin_name}' is not a known Spin command. See spin --help.\n");
```

Comment:
Wondering if `print_error!` might make it clearer that this is a print operation.  This looks too much like `bail!` for me to readily infer that this is a benign piece of user interaction.

Reply:
This is the same as `tracing::error` and `log::error`. How about having the convention of always using this in a qualified way `colors::error!` is hopefully less prone to confusion?

Reply:
That would be better though in that case I'd suggest a namespace like `console` or `terminal` - `tracing` and `log` directly imply output in a way that `colors` (to me) doesn't.

Which might be better because the formatting conventions could come to involve icon-based highlighting or layout functions as well, although I realise this is speculative!

Reply:
I've renamed the dependency to `terminal`.